template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/EmptyWeb-CSharp/Company.WebApplication1.csproj

```diff
@@ -7,6 +7,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Update="Program.Designer.cs">
+      <DependentUpon>Program.cs</DependentUpon>
```

Comment:
xproj used to nest *.Designer.cs files aromatically, but it never made it into CPS-based csproj. Possibly tracked by dotnet/project-system#1272